package/com.unity.formats.usd/Runtime/Scripts/IO/Geometry/MeshExporter.cs

```diff
@@ -295,16 +295,27 @@ public static class MeshExporter {
         scene.Write(path, sample);
         UnityEngine.Profiling.Profiler.EndSample();
 
+        pxr.UsdPrim usdPrim = scene.GetPrimAtPath(path);
+
         // TODO: this is a bit of a half-measure, we need real support for primvar interpolation.
         // Set interpolation based on color count.
         if (sample.colors != null && sample.colors.Length == 1) {
-          pxr.UsdPrim usdPrim = scene.GetPrimAtPath(path);
           var colorPrimvar = new pxr.UsdGeomPrimvar(usdPrim.GetAttribute(pxr.UsdGeomTokens.primvarsDisplayColor));
           colorPrimvar.SetInterpolation(pxr.UsdGeomTokens.constant);
           var opacityPrimvar = new pxr.UsdGeomPrimvar(usdPrim.GetAttribute(pxr.UsdGeomTokens.primvarsDisplayOpacity));
           opacityPrimvar.SetInterpolation(pxr.UsdGeomTokens.constant);
         }
 
+        // for polygonal meshes, we do not want the default subdivision scheme of "catmull-clark" but instead "none"
+        // so that authored normals keep working.
+        var usdGeomMesh = new pxr.UsdGeomMesh(usdPrim);
+        usdGeomMesh.CreateSubdivisionSchemeAttr().Set(new pxr.TfToken("none"));
+
+        // explicitly set the normal interpolation here –
+        // default should be "vertex" anyways but seems some viewers don't respect the default
```

Comment:
Can you note the problematic viewers and versions here, so we can test, for future reference?